src/renderer/views/ProfileSettings/ProfileSettings.js

```diff
@@ -1,27 +1,93 @@
 import { defineComponent } from 'vue'
+import { mapGetters } from 'vuex'
 import FtCard from '../../components/ft-card/ft-card.vue'
 import FtFlexBox from '../../components/ft-flex-box/ft-flex-box.vue'
 import FtProfileBubble from '../../components/ft-profile-bubble/ft-profile-bubble.vue'
 import FtButton from '../../components/ft-button/ft-button.vue'
+import FtLoader from '../../components/ft-loader/ft-loader.vue'
+import FtProfileEdit from '../../components/ft-profile-edit/ft-profile-edit.vue'
+import FtProfileChannelList from '../../components/ft-profile-channel-list/ft-profile-channel-list.vue'
+import FtProfileFilterChannelsList from '../../components/ft-profile-filter-channels-list/ft-profile-filter-channels-list.vue'
+import { calculateColorLuminance, getRandomColor } from '../../helpers/colors'
+import { MAIN_PROFILE_ID } from '../../../constants'
 
 export default defineComponent({
   name: 'ProfileSettings',
   components: {
     'ft-card': FtCard,
     'ft-flex-box': FtFlexBox,
     'ft-profile-bubble': FtProfileBubble,
-    'ft-button': FtButton
+    'ft-button': FtButton,
+    'ft-loader': FtLoader,
+    'ft-profile-edit': FtProfileEdit,
+    'ft-profile-channel-list': FtProfileChannelList,
+    'ft-profile-filter-channels-list': FtProfileFilterChannelsList
+  },
+  data: function () {
+    return {
+      isNewProfileOpen: false,
+      openSettingsProfileId: null,
+      openSettingsProfile: null
+    }
   },
   computed: {
+    ...mapGetters([
+      'profileById'
+    ]),
     profileList: function () {
       return this.$store.getters.getProfileList
+    },
+    isMainProfile: function () {
+      return this.openSettingsProfileId === MAIN_PROFILE_ID
+    }
+  },
+  watch: {
+    profileList: {
+      handler: function () {
+        this.openSettingsProfile = this.getProfileById(this.openSettingsProfileId)
+      },
+      deep: true
     }
   },
+  mounted: function () {
+    this.deletePromptLabel = `${this.$t('Profile.Are you sure you want to delete this profile?')} ${this.$t('Profile["All subscriptions will also be deleted."]')}`
+  },
```

Comment:
Looks like this could be changed to created, which should be a bit more efficient :)
```suggestion
  created: function () {
    this.deletePromptLabel = `${this.$t('Profile.Are you sure you want to delete this profile?')} ${this.$t('Profile["All subscriptions will also be deleted."]')}`
  },
```

Reply:
I copied this part very cautiously from `ProfileEdit.js` when merging it with `ProfileSettings`, because it was not being referenced as a variable, & I thought it was some advanced way we have our `ft-prompt` logic set up, as we have this variable present already in `ft-profile-edit`. but it turns out it was not being used by the code at all, either before or now. Problem solved!